### DISCUSSION TITLE
where are the new raster functions?

### DISCUSSION THREAD

**Discussion by gabrielete** [bytesandbrains/h3-pg#129] [Q&A] [answered]
Hello! I hope you're doing well Zacharias. I just wanted to start by saying how awesome your project is.

I have a database with both the PostGIS extension and the postgis_raster extension installed. Additionally, I have installed h3, and according to "npgx info h3," the version I have installed is 4.1.2. However, when I try to find the new raster functions, such as "\df h3_raster*", it returns nothing.

Could you please guide me on how to resolve this issue? I would greatly appreciate any assistance you can provide.

Thank you so much for your time and for creating such a valuable project.

**Comment by gabrielete** [comment, marked as answer]
Ok, I was missing the h3_postgis extension. 

Thanks anyway!!

**Reply by zachasme** [reply]
You are welcome! It should probably be clearer in the README that there are 2 separate extensions.
